app/assets/config/manifest.js

```diff
@@ -1,4 +1,9 @@
 //= link_tree ../images
+//= link application.scss
 //= link_directory ../stylesheets .css
 //= link_tree ../../javascript .js
 //= link_tree ../../../vendor/javascript .js
+//= jquery
+//= jquery_ujs
+//= popper
+//= bootstrap
```

Comment:
try to move it into `application.js`

Reply:
Already imported to application.js but it didn't work without import into manifest.js

Reply:
okay